src/AzureIoTHub.Portal.Tests.Unit/Client/Pages/EdgeDevices/EdgeDeviceDetailPageTests.cs

```diff
@@ -549,6 +549,7 @@ private IoTEdgeDevice SetupOnInitialisation()
             var mockIoTEdgeDevice = new IoTEdgeDevice()
             {
                 DeviceId = this.mockdeviceId,
+                DeviceName = "test",
```

Comment:
No test is verifying that the field is present on the screen with the expected value. Please add it also.